server/database/modules/index.ts

```diff
@@ -0,0 +1,11 @@
+import donor from './donor';
+import family from './family';
+import donation from './donation';
+import contact from './contact';
+import category from './category';
+import campaign from './campaign';
+import capon from './capon';
+
+export {
+  donor, family, donation, contact, category, campaign, capon,
+};
```

Comment:
You need to rename all your models to start with a capital letter. Also, rename this folder to models instead of modules.